### PR TITLE
Added UI acceptance for the telemetry section 

### PR DESCRIPTION
**Pull request by andr317c**
Our first UI acceptance test for v14 😃 

Added UI acceptance for the telemetry section

### CODE REVIEW

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Telemetry.spec.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  test.afterEach(async ({page, umbracoApi}, testInfo) => {
+    await umbracoApi.telemetry.setTelemetryLevel("Basic");
+  });
+
+  test('can change telemetry level', async ({page, umbracoApi, umbracoUi}) => {
+    const expectedLevel = "Minimal";
+
+    await page.goto(umbracoApi.baseUrl + '/umbraco');
+
+    await page.pause();
```
Comment (Zeegaan) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Telemetry.spec.ts:19]: Don't think we need a page.pause() here 🤣

**Review by Zeegaan — CHANGES_REQUESTED** [review]
Found a small thing

Also why are all the other tests part of this? 🤔 Something is wrong

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Telemetry.spec.ts (hunk 1 above)
Comment (andr317c) [reply tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Telemetry.spec.ts:19]: Whoops, fixed now

Comment (Zeegaan) [reply tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Telemetry.spec.ts:19]: Still not fixed 🤣

**Pull request merged**